jupyterlab.nix

```diff
@@ -54,8 +54,6 @@
         echo jupyterWith needs to build JupyterLab.
         # we need to build the jupyter lab environment before it can be used
         ${jupyterlabEnvBase}/bin/jupyter lab build
-      else
-        echo jupyterWith does not need build JupyterLab. Starting...
```

Comment:
Since some scripts extract information based on Jupyter commands such as <--paths --x>, polluting the raw output info should be unnecessary.

Reply:
>&2 better

Reply:
It's helpful for debugging when things go wrong but don't necessarily error out. What if we tagged it the same way Jupyter does with a `[jupyterWith]` prefix?

Reply:
Something like
```echo "[$(date +'%Y-%m-%d %H:%M:%S') jupyterWith] jupyterWith does not need build JupyterLab. Starting..."```